packages/block-library/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Bug Fix
+
+-   Table block: Background color is not kept ([#39445](https://github.com/WordPress/gutenberg/pull/39445)).
```

Comment:
Nit: Perhaps we could make this a little clearer given the bug is only with named (color palette) colors.
```suggestion
-   Table block: Fix deprecation eligibility to prevent loss of named (color palette) background color ([#39445](https://github.com/WordPress/gutenberg/pull/39445)).
```

Reply:
Thank you for review ! I'll fix this.